pqdb-frontend/src/components/views.js

```diff
@@ -68,30 +288,32 @@ class CustomSQLQuery extends React.Component {
                                         variant="outlined"
```

Comment:
I'd like to autofocus this. Adding `autoFocus` here helps, but only when directly loading the page, not when using the drawer